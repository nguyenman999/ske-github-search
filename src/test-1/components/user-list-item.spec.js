import React from 'react'
import { shallow } from 'enzyme'
import UserListItem from './user-list-item'

describe('user-list-item', () => {
  let props
  beforeEach(() => {
    props = {
      loading: false,
      avatarUrl: 'avatarUrl',
      score: 123,
      login: 'login',
      type: 'type',
    }
  })

  it('render: default', () => {
    const wrapper = shallow(<UserListItem {...props} />)
    expect(wrapper).toMatchSnapshot()
  })
})
