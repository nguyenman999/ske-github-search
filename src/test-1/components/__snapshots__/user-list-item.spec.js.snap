// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user-list-item render: default 1`] = `
<Item>
  <Skeleton
    active={true}
    avatar={true}
    loading={false}
    paragraph={true}
    title={false}
  >
    <Meta
      avatar={
        <Avatar
          shape="circle"
          size="default"
          src="avatarUrl"
        />
      }
      description="type"
      title="login"
    />
    <div>
      123
    </div>
  </Skeleton>
</Item>
`;
